backend/src/entities/song.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Entity, PrimaryGeneratedColumn, Column, OneToOne } from 'typeorm';
+import { Track } from './track.entity';
+
+@Entity()
+export class Song {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  title: string;
+
+  @Column()
+  artist: string;
```

Comment:
Something to improve this definition could be to add unique constraint on song title + artist to avoid duplicates